api/envoy/extensions/filters/http/lua/v3/lua.proto

```diff
@@ -26,7 +27,11 @@ message Lua {
   // further loads code from disk if desired. Note that if JSON configuration is used, the code must
   // be properly escaped. YAML configuration may be easier to read since YAML supports multi-line
   // strings so complex scripts can be easily expressed inline in the configuration.
-  string inline_code = 1 [(validate.rules).string = {min_len: 1}];
+  //
+  // This field is deprecated. Please use
+  // :ref:`default_source_code <envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.default_source_code>`
```

Comment:
better to document the conflict behavior when both inline_code and default_source code specified also